db/migrate/20180924231236_add_disability_compensation_worklow_fields.rb

```diff
@@ -0,0 +1,8 @@
+class AddDisabilityCompensationWorklowFields < ActiveRecord::Migration
+  def change
+    add_column :disability_compensation_submissions, :has_uploads, :boolean
+    add_column :disability_compensation_submissions, :uploads_success, :boolean
+    add_column :disability_compensation_submissions, :has_form_4142, :boolean
```

Comment:
@kreek : In the future for any new document type / forms, are you thinking we would add new columns to the table, for example upcoming forms 781, 8940. 

Just some initial comment for extensibility. Have you given any thought of capturing "doc_type", "form_id" and "upload_status" columns in a new table joined by possibly "va526ez_submit_transaction_id " and "disability_compensation_id" columns?

Reply:
👍Good call. I thought 781 and 8940 were the extent of the ancillary items. Checked with Dawn and there's a least a dozen more ancillary forms 😬. It does mean uploads would then be the outlier with a different pattern. Looking at the fields again we may not actually need the `has_uploads` and `has_form_4142` etc. fields as they could be inferred through the form data in the referenced saved claim table.